src/main/java/me/noodles/gui/commands/Punish.java

```diff
@@ -19,7 +19,7 @@
 	public static String bannedPlayer;
 
 	public Punish() {
-		this.bannedPlayer = null;
+		bannedPlayer = null;
```

Comment:
Static reference fix sneaked in there as well.

Reply:
Possible race-condition?

Reply:
Yep, it's a race-condition bug, when I opened the inventory the second time, the ```bannedPlayer``` gets overwritten.

Reply:
Starting a new PR after this one to fix how we handle banned players.